delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphViewRejection.scala

```diff
@@ -118,34 +115,10 @@ object BlazegraphViewRejection {
       )
 
   /**
-    * Rejection when attempting to decode an expanded JsonLD as an BlazegraphViewValue.
-    *
-    * @param error
-    *   the decoder error
-    */
-  final case class DecodingFailed(error: JsonLdDecoderError) extends BlazegraphViewRejection(error.getMessage)
-
-  /**
-    * Signals an error converting the source Json document to a JsonLD document.
-    */
-  final case class InvalidJsonLdFormat(id: Option[Iri], rdfError: RdfError)
-      extends BlazegraphViewRejection(
-        s"The provided Blazegraph view JSON document ${id.fold("")(id => s"with id '$id' ")}cannot be interpreted as a JSON-LD document."
-      )
-
-  /**
-    * Rejection returned when attempting to create an BlazegraphView where the passed id does not match the id on the
-    * source json document.
-    *
-    * @param id
-    *   the view identifier
-    * @param sourceId
-    *   the view identifier in the source json document
+    * Rejection returned when attempting to decode an expanded JsonLD as an BlazegraphViewValue.
     */
-  final case class UnexpectedBlazegraphViewId(id: Iri, sourceId: Iri)
-      extends BlazegraphViewRejection(
-        s"The provided Blazegraph view '$id' does not match the id '$sourceId' in the source document."
-      )
+  // TODO Remove when the rejection workflow gets refactored / when view endpoints get separated
+  final case class BlazegraphDecodingRejection(error: JsonLdRejection) extends BlazegraphViewRejection(error.reason)
```

Comment:
Needed to be able to reject and try the different view types as they share the same endpoint